### ISSUE TITLE
Camera Housing

### ISSUE THREAD

**Issue by ebgoldstein** [UNCG-DAISY/SediNetCam#5]
design/build/print housing for camera.

keep .stl in the public github repository (`/hardware` directory). make sure to commit as soon as you make changes

**Comment by jacobstasiewicz** [comment]
optimal camera housing will be made after #7  and #6 are solved

**Comment by jacobstasiewicz** [comment]
started on camera housing now that #7 and #6 are done

**Comment by jacobstasiewicz** [comment]
3d printing case for lcd screen

**Comment by jacobstasiewicz** [comment]
Case is done, not printed yet but done

**Comment by jacobstasiewicz** [comment]
Im going to wait to close this till everything is printed and we have put it all together with 0 issues

**Comment by ebgoldstein** [comment]
good idea

**Comment by ebgoldstein** [comment]
i think we can close this for now, I'll make a new issue for a new case
